src/patternfly/base/tokens/tokens-redhat-highcontrast.scss

```
@@ -0,0 +1,758 @@
+  --pf-t--global--breakpoint--350: 60rem;
+  --pf-t--global--breakpoint--400: 62rem;
+  --pf-t--global--breakpoint--500: 75rem;
+  --pf-t--global--breakpoint--550: 80rem;
+  --pf-t--global--breakpoint--600: 90.625rem;
+  --pf-t--global--color--nonstatus--green--400: #3d7019;
+  --pf-t--global--color--status--success--150: #3d7019;
+  --pf-t--global--focus-ring--position--inset: -4px;
+  --pf-t--global--focus-ring--position--offset: 2px;
+  --pf-t--global--focus-ring--width--inset: 3px;
+  --pf-t--global--focus-ring--width--offset: 2px;
+  --pf-t--global--font--family--100: "Red Hat Text", "RedHatText", "Noto Sans Arabic", "Noto Sans Hebrew", "Noto Sans JP", "Noto Sans KR", "Noto Sans Malayalam", "Noto Sans SC", "Noto Sans TC", "Noto Sans Thai", Helvetica, Arial, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)